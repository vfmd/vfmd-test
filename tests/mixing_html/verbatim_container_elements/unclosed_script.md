Unclosed <script> tag

### This is a valid header
