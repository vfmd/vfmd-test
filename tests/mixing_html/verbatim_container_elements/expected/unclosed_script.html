Unclosed <script> tag

<h3>This is a valid header</h3>
